emcc.py

```diff
@@ -1357,7 +1363,7 @@ def save_intermediate(name=None, suffix='js'):
 
     # Optimize, if asked to
     if not LEAVE_INPUTS_RAW:
-      link_opts = [] if debug_level >= 4 else ['-strip-debug'] # remove LLVM debug if we are not asked for it
+      link_opts = [] if debug_level >= 4 or shared.Settings.CYBERDWARF else ['-strip-debug'] # remove LLVM debug if we are not asked for it
```

Comment:
perhaps CYBERDWARF should set debug_level to 4 (which would avoid this check here)? or is there some reason not to do that?